transcom/mymove
# B 20723 int

## [B-20723](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A983376)
## [B-20726](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A983385)

## Summary

Updating PrimeEstimatedWeight to be equal to NTSRecordedWeight for NTS-Release shipments. The NTSRecordedWeight is only available to update for the TOO via CreateMTOShipment or UpdateMTOShipment. In both cases the PrimeEstimatedWeight is set equal to the NTS-Release value once the value exists.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the Office application
2. Login as a TOO
3. Select a move
4. Add an NTS-Release shipment with the NTSRecordedWeight value populated. 
5. Verify the PrimeEstimatedWeight is set equal to the NTSRecordedWeight value in the database.
6. Update either the newly created NTS-Release or another one, preferably with a new NTSRecordedWeight value to ensure the update to PrimeEstimatedWeight is working properly. 
7. Verify the PrimeEstimatedWeight is updated to the (new) NTSRecordedWeight value in the database.

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
